userappwithbroker/build.gradle

```diff
@@ -70,7 +70,7 @@ dependencies {
     implementation "com.google.android.material:material:$rootProject.ext.materialVersion"
 
     localImplementation(project(':adal'))
-    distImplementation("com.microsoft.aad:adal:3.0.2") {
+    distImplementation("com.microsoft.aad:adal:3.1.0") {
```

Comment:
Did we want 3.1.0 here or 3.1.1?

Reply:
Should be 3.1.1 -- thanks, will update

Reply:
Actually, let's hold off on this until we get these bits deployed -- this artifact isn't going to resolve until after we deploy.